src/planner/operator/logical_insert.cpp

```diff
@@ -3,12 +3,48 @@
 namespace duckdb {
 
 void LogicalInsert::Serialize(FieldWriter &writer) const {
-	throw NotImplementedException(LogicalOperatorToString(type));
+	writer.WriteField<idx_t>(insert_values.size());
+	for (auto &entry : insert_values) {
+		writer.WriteSerializableList(entry);
+	}
```

Comment:
@Mytherin does DuckDB's coding style call for a space after a block? (AFAIK clang-format doesn't enforce that?)
Same question with not ending a block with an empty line :-)

Reply:
No strong opinions from my side - I would say anything not enforced by clang-tidy is personal preference/what looks better in that specific situation